Answers.md

```diff
@@ -2,10 +2,22 @@
 
 1.  What is React JS and what problems does it try and solve?
 
-1.  What does it mean to _think_ in react?
+        React is a JavaScript library for building user interfaces. React’s simplicity lies in that it makes it easy to declare user-interfaces in self-contained independent components. In React, every UI is composed of components, so there is no mismatch of multiple types to worry about. Since there is only one common object, it makes it easy to nest components inside components and encourages the principle of single responsibility for each component.
 
-1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+1)  What does it mean to _think_ in react?
 
-1.  Describe state.
+        One of the many great parts of React is how it makes you think about apps as you build them. First of all, React makes you think about each part of the website as component and subcomponent. Secondly, you should make static version of app in React. Than you should start to think about state. State of the app should be allways be in the main container. Lastly, you should add data.

```

Comment:
I think, I should clarify this `state of the app should be always be in the main container` This is NOT always the case but it's the way Gabe describes it and How I like to do it. 


There's nothing set in stone., here.